### ISSUE TITLE
Notable doesn't start on Fedora 35

### ISSUE THREAD

**Issue by DaVukovic** [notable/notable#1602]
- **OS Version**:
Fedora 35
XFCE
- **Notable Version**:
1.8.3.*
### Current behavior
Notable doesn't start. I simply downloaded the rpm-package, installed it and tried to run it. 

Running from the command line gave me some output:

```
Fontconfig warning: "/usr/share/fontconfig/conf.avail/05-reset-dirs-sample.conf", line 6: unknown element "reset-dirs"
[54629:0403/144107.337633:FATAL:gpu_data_manager_impl_private.cc(986)] The display compositor is frequently crashing. Goodbye.
[1]    54629 trace trap (core dumped)  notable

```

Please let me know if you need any other logs. I've had no issue running Notable on Fedora 34 with the very same machine. 

### Expected behavior
I expected it to start.

**Comment by DaVukovic** [comment]
Thanks for helping

I did the changes mentioned above, restarted the machine, but Notable still fails for the same issue

![image](https://user-images.githubusercontent.com/7235787/161722883-b94961bc-ce6c-48a0-8219-acf3233ad742.png)

**Comment by DaVukovic** [comment]
Thanks at least for trying. Maybe we'll find the solution. It would be awesome as this app is one of the best from my POV and for my usecase. 

Hard to find something else.

**Comment by DaVukovic** [comment]
@Alexhuszagh 

Did you find the time to play with it?

**Comment by Alexhuszagh** [comment]
@DaVukovic Yeah, I have. Unfortunately, and I know this is the **least** helpful advice and I'd be happy to try to debug further, but it doesn't replicate on my machine.

Sorry for the small display sizes, but here's just showing it works.
![Screenshot_2022-04-13_10-06-14](https://user-images.githubusercontent.com/9440903/163211423-33fc01a5-95de-4105-aeb6-84e472221ec8.png)
![Screenshot_2022-04-13_10-06-36](https://user-images.githubusercontent.com/9440903/163211427-ec699110-3aec-44f8-a6b4-6b5c305e10a5.png)

If it is a fontconfig issue, then there should be 2 files that you need to remove to go back to system fonts. Note that this is destructive, and you might lose any custom installed fonts I believe, so definitely make backups of the files.

```bash
mv ~/.fonts.conf ~/.fonts.conf.bak
mv ~/.config/fontconfig/fonts.conf ~/.config/fontconfig/fonts.conf.bak
```

And then reinstall fontconfig, reboot your machine, and see if that works. I should stress: this is destructive, so maybe export any custom fonts you have prior to doing this.

```bash
sudo dnf reinstall fontconfig
```

**Comment by MMK21Hub** [comment]
> Or is it to be expected that this software is now unmaintained as the last changes have been done over a year ago?
@DaVukovic Have you checked out [the development builds](https://github.com/notable/notable-insiders/releases)? All the development is happening over there.

**Comment by ThumusLive** [comment]
I have the same issue but on ubuntu
![image](https://user-images.githubusercontent.com/42680097/168929578-de2be4aa-6be4-4a1a-ae65-f180744d11cc.png)
I think it's not link to fedora or ubuntu directly

**Comment by Ahryman40k** [comment]
Same problem here on Garuda Linux (Arch based)

`Fontconfig warning: "/usr/share/fontconfig/conf.avail/05-reset-dirs-sample.conf", line 6: unknown element "reset-dirs"
[311288:0606/100339.386055:FATAL:gpu_data_manager_impl_private.cc(986)] The display compositor is frequently crashing. Goodbye.
fish: Job 1, './Notable-1.8.4.AppImage' terminated by signal SIGTRAP (Trace or breakpoint trap)`

I will test with the latest beta

**Comment by DaVukovic** [comment]
@Alexhuszagh Thanks for trying. Much appreciated. I haven't customized my compositor at all. Moving those files, rebooting the machine and reinstalling fontconfig didn't help. 

@MMK21Hub Thanks for suggesting the beta channel. Unfortunately there's no rpm package available. I'm not going to use Snap and/or Flatpak as I simply don't like it. 

By googling "Fontconfig reset dirs" I get a bunch of ppl claiming about this issue and that it seems to be the task of the application to get this fixed. 

Don't get me wrong. I hope I don't sound ungrateful. But there's nothing what I could either do or test. As long as there's no rpm package available for the beta releases, this bug still stands. And obviously it's not a Fedora only issue. So let's hope this will be fixed at some point. 

At least it works well on my Mac.

**Comment by avila** [comment]
Same here on Fedora 36, affecting both the Appimage as well the RPM package. Same `Fontconfig warning` and compositor crashing.

**Comment by eclectic-kitty** [comment]
I'm having the same issue on Pop OS 22.04. No fontconfig warning, but same thing with the compositor.

**Comment by ewuerger** [comment]
Almost one year later, and I also have the same behavior. PopOS 22.04.

**Comment by GhostDog98** [comment]
Still having this behaviour arch linux...

**Comment by gStart9** [comment]
Almost 4 years later, on Notable 1.8.4 (.deb and .AppImage) on Debian Trixie x86_64, same issue.

Moving the ~/.cache/fontconfig dir and rebooting didn't help.
